Alerting: Add a new plugin for jira service management (#78275)
grafana/grafana: docs/sources/introduction/_index.md

## Alerts

> If you're using Grafana Alerting, then you can have alerts sent through a number of different alert notifiers, including PagerDuty, SMS, email, VictorOps, OpsGenie, Jira Service Management or Slack.

Reviewer: ```suggestion
Grafana Alerting can send alerts through several different alert notifiers, including PagerDuty, SMS, email, VictorOps, OpsGenie, Jira Service Management, or Slack.
```
